bears/natural_language/SpellCheckBear.py

```diff
@@ -10,7 +10,7 @@ class SpellCheckBear:
     """
     Lints files to check for incorrect spellings using ``scspell``.
```

Comment:
thats good, maybe if you could change the commit body "Update -> Updates" would be amazing :D